openedx/adg/common/course_meta/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+This app adds the ability to create and manage metadata for courses, without making any core changes.
+
+Currently course metadata contains course short id only. It is a 4 digit id for a course to sync its
```

Comment:
Can we please rephrase it in a way that in future if we would add more setting or change settings, chances of the need to update documentation should me minimal? For example instead of say currently it only contains .... We can say, "For example, course_short_id, which is being used for this"